packages/plugins/integration-zapier-ui/src/lib/components/zapier-actions/zapier-actions.component.html

```
@@ -18,7 +18,7 @@ <h6>{{ action.name }}</h6>
                 <p>{{ action.description }}</p>
               </div>
               <div class="endpoint-actions">
-                <button nbButton status="primary" size="small" outline (click)="openActionDetails()">
+                <button nbButton status="primary" size="small" outline (click)="openActionDetails(action)">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)